Files/ViewModels/PreviewPaneViewModel.cs

```diff
@@ -234,18 +242,8 @@ public async void UpdateSelectedItemPreview(bool downloadItem = false)
                     // If that fails, revert to no preview/details available as long as the item is not a shortcut or folder
                     if (SelectedItem != null && !SelectedItem.IsShortcutItem && SelectedItem.PrimaryItemAttribute != StorageItemTypes.Folder)
                     {
-                        try
-                        {
-                            var basicModel = new BasicPreviewViewModel(SelectedItem);
-                            await basicModel.LoadAsync();
-                            PreviewPaneContent = new BasicPreview(basicModel);
-                            PreviewPaneState = PreviewPaneStates.PreviewAndDetailsAvailable;
-                            return;
-                        }
-                        catch (Exception ex)
-                        {
-                            Debug.WriteLine(ex);
-                        }
+                        await LoadBasicPreviewAsync();
```

Comment:
Perhaps we can revert this change: if there's an exception loading the basic preview we should execute `PreviewPaneContent = null;` and not return, right?